pontoon/base/models.py

```diff
@@ -1190,6 +1316,18 @@ def aggregate_stats(self, instance):
 
         instance.save()
 
+    def resource_locale_stats(self, paths, locale):
```

Comment:
You should rename this method to just `stats`, because TranslatedResource already defines resource and locale.

And move it under the `AggregatedStats` as the model method.